qa/rpc-tests/p2p-versionbits-warning.py

```diff
@@ -112,27 +113,39 @@ def run_test(self):
 
         # 1. Have the node mine one period worth of blocks
         self.nodes[0].generate(VB_PERIOD)
+        assert(self.nodes[0].getblockcount() ==  144)
 
         # 2. Now build one period of blocks on the tip, with < VB_THRESHOLD
         # blocks signaling some unknown bit.
         nVersion = VB_TOP_BITS | (1<<VB_UNKNOWN_BIT)
-        self.send_blocks_with_version(test_node, VB_THRESHOLD-1, nVersion)
+        #self.send_blocks_with_version(test_node, VB_THRESHOLD-1, nVersion)
```

Comment:
Please do not comment out code you want to remove. The old code remain in source control history so this is not necessary.